### PR TITLE
Adding feature that gets/sets exerise config in JSON or Yaml format.

### PR DESCRIPTION
**Pull request by krulis-martin**

### CODE REVIEW

File: recodex/plugins/exercises/cli.py [hunk 1]
```diff
@@ -108,3 +110,35 @@ def check_rs_evaluations(api: ApiClient, threshold):
...
+@click.argument("exercise_id")
+@click.option("--json/--yaml", "useJson", default=True)
+@pass_api_client
+def get_config(api: ApiClient, exercise_id, useJson):
+    """
+    Get exercise configuration in JSON (or possibly yaml) format
+    """
+    config = api.get_exercise_config(exercise_id)
+    if useJson:
+        click.echo(json.dumps(config, sort_keys=True, indent=4))
```
Comment (SemaiCZE) [inline recodex/plugins/exercises/cli.py]: Why not use `json.dump()` to keep it similar with YAML version?

File: setup.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(name='recodex-cli',
-      version='0.0.7',
+      version='0.0.9',
```
Comment (janbuchar) [inline setup.py:4]: How does this feature deserve two minor versions instead of one? :)

File: recodex/plugins/exercises/cli.py (hunk 1 above)
Comment (janbuchar) [reply recodex/plugins/exercises/cli.py]: I'm with @SemaiCZE here - there's no need for `click.echo` here.

Comment (krulis-martin) [reply recodex/plugins/exercises/cli.py]: Sure thing.

File: setup.py (hunk 2 above)
Comment (krulis-martin) [reply setup.py:4]: Actually, we already have 0.0.8, version in Git was not updated.

**Review by SemaiCZE — APPROVED** [review]

**Pull request merged**